src/dtls13.c

```diff
@@ -1554,6 +1558,17 @@ static int _Dtls13HandshakeRecv(WOLFSSL* ssl, byte* input, word32 size,
 
     isFirst = fragOff == 0;
     isComplete = isFirst && fragLength == messageLength;
+
+    if (!isComplete && (handshakeType == client_hello ||
```

Comment:
maybe it's better to check epoch 0 here?

Reply:
Using `!IsEncryptionOn(ssl, 0)` instead to check for plaintext messages.